lib/cucumber/core/test/result.rb

```diff
@@ -177,6 +177,24 @@ def ok?(be_strict = false)
           end
         end
 
+        class Ambiguous < Raisable
+          include Result.query_methods :ambiguous
+
+          def describe_to(visitor, *args)
+            visitor.failed(*args)
```

Comment:
The ambiguous result should describe itself as `ambiguous` to the visitors (which also requires and `ambiguous` method in the [`RunningTestCase`](https://github.com/cucumber/cucumber-ruby-core/blob/master/lib/cucumber/core/test/runner.rb#L82) - same definition as `undefined` works)